### ISSUE TITLE
front: responsiveness problem on timetable stops

### ISSUE THREAD

**Issue by axrolld** [OpenRailAssociation/osrd#5494]
### What happened?

![image](https://github.com/osrd-project/osrd/assets/86296016/cc588789-a215-429c-99fb-4553bc254c6f)

### How can we reproduce it (as minimally and precisely as possible)?

1. Create a train with one stop
2. Shorten your window size

### What operating system, browser and environment are you using?

- **Env**: rec-osrd.reseau.sncf.fr

### OSRD version (top right corner `Account` button > `Informations`)

1482e8c

**Comment by tykayn** [comment]
@axrolld i cannot reproduce the superposition of texts locally and on staging, can you still reproduce it ?
https://osrd.staging.dgexsol.fr/operational-studies/projects/10/studies/15/scenarios/29
![Screenshot 2023-12-11 at 14-42-09 OSRD](https://github.com/osrd-project/osrd/assets/3931245/23190625-3b12-45b5-b972-cef920d38ccf)

**Comment by nicolaswurtz** [comment]
Yes it's reproducible but as shown in Alex's screenshot, you have to have a stop, 2 times, arrival & departure.

I suggest to correct it with a fixed width for time, and allow overflow for the entire traindriver schedule.

**Comment by SharglutDev** [comment]
The bug seems to happen only if there is a stop on a "+" PR like `Marmande 10:47+`

**Comment by axrolld** [comment]
Still present

**Comment by SharglutDev** [comment]
@axrolld if I understood correctly, this table will soon be removed in favor of exporting a simulation sheet and csv (see [mockup](https://www.sketch.com/s/2fb064cd-c483-46ac-bd30-72705fd63e75/a/zyYkygo#Inspect)).
Should we still maintain this component ?

**Comment by Synar** [comment]
As I believe the table has now been removed, should the issue be closed as not planned?

**Comment by SharglutDev** [comment]
Yes totally :)
